src/pages/AddPurchase/AddPurchase.js

```diff
@@ -0,0 +1,133 @@
+import FoodList from '../../components/FoodList/FoodList'
+import SearchFood from '../../components/SearchFood/SearchFood'
+import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import getFood from '../../services/getFood'
+import SumFootprint from '../../components/SumFootprint/SumFootprint'
+import PurchaseList from '../../components/PurchaseList/PurchaseList'
+import loadLocally from '../../lib/loadLocally'
+import saveLocally from '../../lib/savelocally'
+
+export default function AddPurchase() {
+  const [foodList, setFoodList] = useState([])
+  const [searchInput, setSearchInput] = useState('')
+  const [purchasedFood, setPurchasedFood] = useState(
+    loadLocally('purchasedFood') ?? []
+  )
+  const [foodListModal, setFoodListModal] = useState(false)
+  const [carbonFootprintSum, setCarbonFootprintSum] = useState(
+    loadLocally('carbonFootprintSum') ?? 0
+  )
+  const [pointerPosition, setPointerPosition] = useState(0)
+
+  useEffect(() => {
+    getFood()
+      .then((data) => setFoodList(data))
+      .catch((error) => console.log(error))
+  }, [])
+
+  function handleChange(event) {
```

Comment:
This function name is very generic, consider a more telling name (e.g. `handleSearchInput`? )